app/javascript/components/widget-controller.jsx

```diff
@@ -72,6 +73,9 @@ export const WidgetController = ({ client, cityName }) => {
           console.error(error);
           return <DisconnectedDisplay cityName={cityName} error={error} />;
         }
+        if (data.location.widgets.enabled === []) {
+          return <WidgetDisabledDisplay />;
```

Comment:
:thinking: Can we ever leave the `WidgetDisabledDisplay` path if widgets are enabled for the city?